themes/src/main/resources/theme/keycloak-preview/account/resources/systemjs.config.js

```diff
@@ -88,11 +88,19 @@
               './Card/CardBody': '@empty',
               './Card/CardHeader': '@empty',
               './Card/CardFooter': '@empty',
-              './Checkbox': '@empty', //'./Checkbox/index.js',
+              './Checkbox': './Checkbox/index.js', //'./Checkbox/index.js',
               './ChipGroup': '@empty', //'./ChipGroup/index.js',
               './ClipboardCopy': '@empty', //'./ClipboardCopy/index.js,
               './ContextSelector': '@empty', //'./ContextSelector/index.js',
-              './DataList': '@empty', //'./DataList/index.js',
+              './DataList': './DataList/index.js', 
+              './DataList/DataListItem': './DataList/DataListItem.js', 
+              './DataList/DataListItemRow': './DataList/DataListItemRow.js', 
+              './DataList/DataListCell': './DataList/DataListCell.js', 
+              './DataList/DataListCheck': './DataList/DataListCheck.js', 
+              './DataList/DataListAction': './DataList/DataListAction.js',
+              './DataList/DataListToggle': './DataList/DataListToggle.js', 
+              './DataList/DataListContent': './DataList/DataListContent.js',
+              './DataList/DataListItemCells': './DataList/DataListItemCells.js', 
```

Comment:
You only need ./DataList/index.js.  You shouldn't need an entry for each component because index.js already takes care of those.